var/spack/repos/builtin/packages/ecp-data-vis-sdk/package.py

```diff
@@ -87,26 +87,29 @@ class EcpDataVisSdk(BundlePackage, CudaPackage, ROCmPackage):
     variant("vtkm", default=False, description="Enable VTK-m")
     variant("zfp", default=False, description="Enable ZFP")
 
+    # Language Options
+    variant("fortran", default=True, sticky=True, description="Enable fortran language features.")
+
     ############################################################
     # Dependencies
     ############################################################
     cuda_arch_variants = ["cuda_arch={0}".format(x) for x in CudaPackage.cuda_arch_values]
     amdgpu_target_variants = ["amdgpu_target={0}".format(x) for x in ROCmPackage.amdgpu_targets]
 
     dav_sdk_depends_on(
-        "adios2+shared+mpi+fortran+python+blosc+sst+ssc+dataman",
+        "adios2+shared+mpi+python+blosc+sst+ssc+dataman",
         when="+adios2",
-        propagate=["hdf5", "sz", "zfp"],
+        propagate=["hdf5", "sz", "zfp", "fortran"],
     )
 
     dav_sdk_depends_on("darshan-runtime+mpi", when="+darshan", propagate=["hdf5"])
     dav_sdk_depends_on("darshan-util", when="+darshan")
 
     dav_sdk_depends_on("faodel+shared+mpi network=libfabric", when="+faodel", propagate=["hdf5"])
 
-    dav_sdk_depends_on("hdf5@1.12: +shared+mpi+fortran", when="+hdf5")
+    dav_sdk_depends_on("hdf5@1.12: +shared+mpi", when="+hdf5", propagate=["fortran"])
```

Comment:
Similarly, I was expecting something along the lines:

```suggestion
    dav_sdk_depends_on("hdf5@1.12: +shared+mpi", when="+hdf5")
    dav_sdk_depends_on("hdf5@1.12: +shared+mpi+fortran", when="+hdf5+fortran", propagate=["fortran"])
```

Reply:
And the same approach for line 112 and 164-165 below.